tests/ClientTest.php

```diff
@@ -0,0 +1,252 @@
+<?php 
+
+namespace Aerospike;
+use PHPUnit\Framework\TestCase;
+
+final class ClientTest extends TestCase
+{
+    protected static $cp;
+    protected static $client;
+    protected static $key;
+
+    protected static $namespace = "test";
+    protected static $set = "test";
+
+
+    public static function setUpBeforeClass(): void
+    {
+        self::$cp = new ClientPolicy();
+
+        try {
+            self::$client = Aerospike(self::$cp, "127.0.0.1:3000");
```

Comment:
Probably a good idea to have these settings as constants as well. Even better would be a solution to pass them on the CLI as parameters.